### PR TITLE
Added more info on appgw in spoke vnet

### PR DESCRIPTION
**Pull request by froand**
Added some more updated and descriptive guidance from the recently released Zero-trust network for web applications from Jose.

### CODE REVIEW

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[5f821fa](https://github.com/froand/cloud-adoption-framework/commits/5f821fafb0db721d59a89d4d915e8abb5ded940f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/ready/azure-best-practices/plan-for-app-delivery.md](https://github.com/froand/cloud-adoption-framework/blob/patch-1/docs/ready/azure-best-practices/plan-for-app-delivery.md) | :warning:Warning | | [Details](#user-content-3c64205147a51d95281bb1f2fc9233fd-a0d99d8b)

<a id="3c64205147a51d95281bb1f2fc9233fd-a0d99d8b"></a>
### [docs/ready/azure-best-practices/plan-for-app-delivery.md](https://github.com/froand/cloud-adoption-framework/blob/patch-1/docs/ready/azure-best-practices/plan-for-app-delivery.md)
 - **Line 36, Column 77**: **[Warning: hard-coded-locale - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#application-gateway-limits' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 37, Column 29**: **[Warning: hard-coded-locale - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://learn.microsoft.com/en-us/azure/architecture/example-scenario/gateway/application-gateway-before-azure-firewall' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 36, Column 77**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#application-gateway-limits' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 37, Column 29**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/en-us/azure/architecture/example-scenario/gateway/application-gateway-before-azure-firewall' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202211090724413239-995/BuildReport?accessString=68858a3b5dc7edc2716478f0e266578e8cac0f7d9f4c8b3d2cadc9cc97b23f4b).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by ShannonLeavitt** [comment]
@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers - Please review the proposed changes. If you want to move forward with the content, add a sign-off.

@MicrosoftDocs/public-repo-pr-review-team

#assign: @martinekuan
#unassign: @BrianBlanchard

File: docs/ready/azure-best-practices/plan-for-app-delivery.md [hunk 1]
```diff
@@ -26,7 +26,15 @@ This section explores key recommendations to deliver internal-facing and externa
...
+- Perform application delivery within landing zones for both internal-facing and external-facing applications. 
+  - Treat the Application Gateway as an application component and deploy it in a spoke virtual network not as a shared resource in the hub.
+
+  - It can be difficult to troubleshoot Web Application Firewall alerts. You generally need in-depth knowledge of the application to decide whether the messages that trigger those alarms are legitimate. 
+
+  - You might face role-based access control problems if you deploy Application Gateway in the hub when teams manage different applications but use the same instance of Application Gateway. Each team then has access to the entire Application Gateway configuration.
+
+  - If you treat Application Gateway as a shared resource, you might exceed [Azure Application Gateway limits](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#application-gateway-limits)
+  - Read more about this in [Zero-trust network for web applications](https://learn.microsoft.com/en-us/azure/architecture/example-scenario/gateway/application-gateway-before-azure-firewall)
```
Comment (ckittel) [inline docs/ready/azure-best-practices/plan-for-app-delivery.md:37]:
```suggestion
  - Treat the Application Gateway as an application component and deploy it in a spoke virtual network not as a shared resource in the hub.
  - To interpret Web Application Firewall alerts, you generally need in-depth knowledge of the application to decide whether the messages that trigger those alerts are legitimate.
  - You might face role-based access control problems if you deploy Application Gateway in the hub when teams manage different applications but use the same instance of Application Gateway. Each team then has access to the entire Application Gateway configuration.
  - If you treat Application Gateway as a shared resource, you might exceed [Azure Application Gateway limits](/azure/azure-resource-manager/management/azure-subscription-service-limits#application-gateway-limits)
  - Read more about this in [Zero-trust network for web applications](/azure/architecture/example-scenario/gateway/application-gateway-before-azure-firewall)
```

**Review by ckittel — APPROVED** [review]
I'm a huge fan of this guidance.

**Comment by ckittel** [comment]
CACT #sign-off

File: docs/ready/azure-best-practices/plan-for-app-delivery.md [hunk 2]
```diff
@@ -26,7 +26,12 @@ This section explores key recommendations to deliver internal-facing and externa
...
 
-- Perform application delivery within landing zones for both internal-facing and external-facing applications.
+- Perform application delivery within landing zones for both internal-facing and external-facing applications. 
+  - Treat the Application Gateway as an application component and deploy it in a spoke virtual network not as a shared resource in the hub.
+  - To interpret Web Application Firewall alerts, you generally need in-depth knowledge of the application to decide whether the messages that trigger those alerts are legitimate.
+  - You might face role-based access control problems if you deploy Application Gateway in the hub when teams manage different applications but use the same instance of Application Gateway. Each team then has access to the entire Application Gateway configuration.
+  - If you treat Application Gateway as a shared resource, you might exceed [Azure Application Gateway limits](/azure/azure-resource-manager/management/azure-subscription-service-limits#application-gateway-limits)
+  - Read more about this in [Zero-trust network for web applications](/azure/architecture/example-scenario/gateway/application-gateway-before-azure-firewall)
```
Comment (Jak-MS) [inline docs/ready/azure-best-practices/plan-for-app-delivery.md:34]:
```suggestion
  - If you treat Application Gateway as a shared resource, you might exceed [Azure Application Gateway limits](/azure/azure-resource-manager/management/azure-subscription-service-limits#application-gateway-limits).
  - Read more about this in [Zero-trust network for web applications](/azure/architecture/example-scenario/gateway/application-gateway-before-azure-firewall).
```

**Pull request merged**